sos/report/plugins/postgresql.py

```diff
@@ -85,34 +83,14 @@ def setup(self):
         self.add_cmd_output("du -sh %s" % self.get_option('pghome'))
 
 
-class RedHatPostgreSQL(PostgreSQL, SCLPlugin):
-
-    packages = (
-        'postgresql',
-        'rh-postgresql95-postgresql-server',
-        'rh-postgresql10-postgresql-server',
-        'rh-postgresql12-postgresql-server',
-    )
```

Comment:
We should not introduce any kind of regression by dropping `postgres-server` from `packages` here. Since `postgresql-server` package has a dependency on `postgresql` package already. ACK.